contracts/repository/blockchain/casino.go

```diff
@@ -170,6 +171,112 @@ func (r *CasinoBlockchainRepo) GetBonusBalances(casinos []*models.Casino, accoun
 	return bonusBalances, nil
 }
 
+func (r *CasinoBlockchainRepo) getTokenContractBalances(tokenContract eos.AccountName,
+	playerAccount eos.AccountName) ([]eos.Asset, error) {
+	resp, err := r.bc.Api.GetTableRows(eos.GetTableRowsRequest{
+		Code:  string(tokenContract),
+		Scope: string(playerAccount),
+		Table: "accounts",
+		JSON:  true,
+	})
+	if err != nil {
+		return nil, err
+	}
+	var jsonBalances []struct {
+		Balance eos.Asset `json:"balance"`
+	}
+	if err := resp.JSONToStructs(&jsonBalances); err != nil {
+		return nil, err
+	}
+	balances := make([]eos.Asset, len(jsonBalances))
+	for i, b := range jsonBalances {
+		balances[i] = b.Balance
+	}
+	return balances, nil
+}
+
+type TokenContract struct {
```

Comment:
this struct is used in single function, why don't use anonymous struct?

Reply:
fixed